src/ncp/spinel.c

```diff
@@ -2125,6 +2133,14 @@ const char *spinel_status_to_cstr(spinel_status_t status)
         ret = "STATUS_JOIN_INCOMPATIBLE";
         break;
 
+    case SPINEL_STATUS_JOIN_RSP_TIMEOUT:
```

Comment:
`SPINEL_STATUS_JOIN_RSP_TIMEOUT` -> `SPINEL_STATUS_JOIN_TIMEOUT`?

Reply:
Prefer to keep it as is as it corresponds to `OT_ERROR_RESPONSE_TIMEOUT` in core and more clearly in indicating the interim JOIN_FIN/JOIN_ENT process.